Monika After Story/game/script-songs.rpy

```diff
@@ -1355,3 +1355,37 @@ label mas_monika_plays_or(skip_leadin=False):
     window auto
 
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_songs_database,
+            eventlabel="mas_song_abstract_nonsense",
+            category=[store.mas_songs.TYPE_SHORT],
+            prompt="Abstract Nonsense",
+            random=True,
+            aff_range=(mas_aff.NORMAL,None)
+        ),
+        code="SNG"
+    )
+
+label mas_song_abstract_nonsense:
+    m 2dfd "{i}~This is useless,{/i}{w=0.5}"
+    extend 2rkd "{i} I want it to stop,{/i}{w=0.5}"
+    extend 6rkc "{i} I put my face out the window~{/i}"
+    m 2dfd "{i}~This is boring,{/i}{w=0.5}"
+    extend 2rkd "{i} I'm tired of it,{/i}{w=0.5}"
+    extend 6rkc "{i} but I don't have the courage to die~{/i}"
+    m 2dfd "{i}~This is useless,{/i}{w=0.5}"
+    extend 2rkd "{i} I want it to stop,{/i}{w=0.5}{nw}"
+    extend 5rkc "{i} I'm injecting my wrist~{/i}"
+    m 2dfd "{i}~This is boring,{/i}{w=0.5}"
+    extend 2rkd "{i} I'm tired of it,{/i}{w=0.5}"
+    extend 1tsc "{i} I eat the cake{/i}{w=0.5}"
+    extend 6dsd "{i} and take a deep breath~{/i}"
+    m 1ekd "You know, [player], I really feel bad for the main character of this song..."
+    m 3wkd "And one of the worst parts is that a lot of people really relate to how she feels."
+    m 6dkd "...It really breaks my heart."
+    m 1ekd "Please remember that I'll always be here for you and that you're never really alone."
+    m 1dkb "I really do love you."
+    return "derandom" "love"
```

Comment:
```suggestion
    return "derandom|love"
```
For the sake of code consistency. Not sure if return with 2 arguments is actually wrong.